test/test_hag.py

```diff
@@ -3,7 +3,7 @@
 
 
 edge_indexes = torch.randint(0, 100, (2, 100))
-direction = 0
+direction = 1
 print(edge_indexes)
 
 print(graph_to_hag(edge_indexes, direction))
```

Comment:
The tests don't test anything.

IMO, it would be great to have at least 1 test that fails until the `graph_to_hag` works as expected.

Reply:
It isn't a proper test yet. I just use it for the dev of the code.

Reply:
Ok.